site/Dance/hero.php

```diff
@@ -0,0 +1,13 @@
+<div class="header-box">
+      <img class="header-box-image" src="../assets/images/placeholder-image-rectangle.png">
+      <div class="header-box-title">
+        <?php print $headertext; ?>
+      </div>
+    </div>
+    <div>
+      <p class="header-text"><?php print $headertext2; ?></p>
+    </div>
```

Comment:
You're missing a `</div>` here. Easy way to catch this is by looking at the indentation. Typically the last closing `div` in the file should be all the way to the left edge.